src/pspm_write_channel.m

```diff
@@ -95,17 +95,21 @@
         if isfield(newdata{i}, 'data') && isfield(newdata{i}, 'header')
             d = newdata{i}.data;
             [h,w] = size(d);
-            if w ~= 1
+            if h==0 && w==0

```

Comment:
As I understand, "the empty data field" indicates that "newdata{i} has a field data, but it is empty or 0x0", thus I add the update for this situation.